scripts/apps/authoring/styles/themes.scss

```diff
@@ -511,6 +511,14 @@ body, html {
             }
         }
     }
+    .disallowed-char-error {
```

Comment:
I'm not sure why we would need special styles for char error, but I'm pretty sure we don't need it 2x.. :wink:

Reply:
At first, I thought to use the `char-count error` class to make it look like the one shown in the above attachment but later on, decided to make a new one. Using it 2X because slugline could be in `authoring-header` as well as in the `content-fields`.

Reply:
sure, but you don't have to define the class inside any parent to make it work in both